superset/assets/spec/javascripts/modules/utils_spec.jsx

```diff
@@ -10,12 +9,6 @@ import {
 } from '../../../src/modules/utils';
 
 describe('utils', () => {
-  it('slugify slugifies', () => {
```

Comment:
can we still rewrite but keep this test case?

Reply:
I tested stubbing Superset's `slugify` implementation with `kebabCase` from `lodash` and it passes all test cases under this `it` except one. 

`kebabCase`: `"camelCase"` => `"camel-case"`
`slugify`: `"camelCase"` => `"camelcase"`

This was the only difference. IMO, I am leaning towards switching to `lodash`'s `kebabCase` which is well-tested library and industry standard than custom implementation and then remove `slugify` from codebase.

I checked where the function `slugify` is being used and it is used for creating tooltip `id` string. My understanding is this is harmless change, so I made the move for it. Please correct me if I misunderstood.

Reply:
it used in creating customized url. for example `/superset/dashboard/homesmetrics/`.
can you make sure existed slugged dashboard url work?

Reply:
i think url is generated by python slugify function...so this JS version is just "tooltip". so it really didn't matter much.